### PR TITLE
Aes

### PR DESCRIPTION
**Pull request by raidoz**

### CODE REVIEW

File: make/jtags/ft2232d.jtag [hunk 1]
```diff
@@ -55,4 +55,9 @@ erase-sig:
 	openocd -f $(PROGRAMMER_CONFIG) -c "transport select $(PROGRAMMER_TRANSPORT)" -f target/efm32.cfg -c "init" -c "reset halt" -c "mww 0x400e0008 0x01" -c "mww 0x400e0010 0x0FE00000" -c "mww 0x400e000c 0x01" -c "mww 0x400e000c 0x02" -c "reset run" -c "shutdown"
 	ftdiutil -f $(PROGRAMMER_CONFIG) -c "init" -c "ftdi_set_bitmode reset" -c "ftdi_rebind_on_exit"
 
+install-fhead: $(PROGRAM_FHEADER)
```
Comment (DaStoned) [inline make/jtags/ft2232d.jtag]: Recommend PROGRAM_FHEADER_IMG or something similar

File: make/Makerules [hunk 2]
```diff
@@ -128,6 +128,10 @@ endef
 #   @return C-style string with each character as a raw hex byte ("\xNN\xNN...\xNN")
 uuidToCstr = "$(shell echo $(subst -,,$(1)) | $(SED) 's/.\{2\}/\\x&/g')"
 
+# Convert AES-KEY string to a C representation of raw bytes
+#   @param $(1) AES-KEY 16 bytes (nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn)
+#   @return C-style string with each character as a raw hex byte ("\xNN\xNN...\xNN")
+aesToCstr = "$(shell echo $(1) | $(SED) 's/.\{2\}/\\x&/g')"
```
Comment (DaStoned) [inline make/Makerules]: This is a general purpose function for converting any set of bytes properly presented, not only AES keys. Perhaps name it so?

File: make/jtags/ft2232d.jtag (hunk 1 above)
Comment (kobilozor) [reply make/jtags/ft2232d.jtag]: Fixed in commit 1c97ae3e08a7b26ed4601fe9774ebfd9040941fe

File: make/Makerules (hunk 2 above)
Comment (kobilozor) [reply make/Makerules]: Fixed in commit 1c97ae3e08a7b26ed4601fe9774ebfd9040941fe

**Review by DaStoned — APPROVED** [review]

**Pull request merged**